routes/web/hardware.php

```diff
@@ -181,4 +181,8 @@ function () {
             'middleware' => ['auth'],
             'parameters' => ['asset' => 'asset_id'
         ],
-]);
\ No newline at end of file
+]);
+
+Route::get('ht/{any?}',
```

Comment:
What’s `ht` here? Can you clarify what this new route does?

Reply:
maybe this isnt helpful or new info as Im not the OP, but some of the previous comments in the PR mention that this is a shorter version of the existing url endpoint to allow the QR code data to be shorter/smaller/simpler.

Reply:
"ht" stands for Hardware Tag. The route is equivalent to [baseurl]/hardware/bytag/[hardwaretag].
It was created so that URLs can be shorter, and therefore QR codes can be simpler.